site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -82,7 +82,7 @@ class OAuth2Controller(
         withHeaders(HeaderNames.AUTHORIZATION → s"token $authToken").
         get().map { response ⇒
           val login = (response.json \ "login").as[String]
```

Comment:
This is a really bad way to parse de json it should all be marshaled and validated at once, as is it may throw an exception if the incoming messages get malformed due to API changes on the Github side.